src/main/java/dk/cngroup/trainings/spring/springassignment/exception/UserEmailExistsException.java

```diff
@@ -0,0 +1,7 @@
+package dk.cngroup.trainings.spring.springassignment.exception;
+
+public class UserEmailExistsException extends Throwable {
```

Comment:
if you will implement the wrapping of exceptions as said above, you can also review number of exceptions. Maybe having exception for every possible type of error is too much, maybe you can achieve similar results with different approach (having some more fields in the exceptions).

Bottomline: if every exception is handled the same way (just print its message for example), maybe you dont need to have that much exceptions.